packages/docs/src/stories/Button.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { StoryObj, Meta } from '@storybook/react'
+import { Button, ButtonProps } from '@codelife-ui/react'
+import { FaGithub } from 'react-icons/fa'
+import React from 'react'
+export default {
+  title: 'Form/Button',
+  component: Button,
+  tags: ['autodocs'],
+  args: { children: 'Send', size: 'sm' },
+} as Meta<ButtonProps>
```

Comment:
```suggestion
argtypes:{size: {
      options: ['sm', 'md', 'lg', 'xl'],
      control: {
        type: 'select',
      },
    },
  },
} as Meta<ButtonProps>
```
Also can add options to buttons like primary, secondary...

Reply:
Done!